### PR TITLE
[ADF-2432] Creating component to display node permission

### PR DESCRIPTION
**Pull request by VitoAlbano**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [X] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
No component shows the nodes permission

**What is the new behaviour?**
Display node permission component is created

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: docs/content-services/permission-display.component.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+Added: v2.3.0
+Status: Active
+Last reviewed: 2018-03-12
+---
+
+# Permission Display Component
```
Comment (eromano) [inline docs/content-services/permission-display.component.md]: can we add a screenshot?

File: lib/content-services/index.ts [hunk 2]
```diff
@@ -31,6 +31,8 @@ export * from './dialogs/dialog.module';
 export * from './folder-directive/folder-directive.module';
 export * from './content-metadata/content-metadata.module';
 export * from './directives/content-directive.module';
+export { NodeDownloadDirective } from './directives/node-download.directive';
```
Comment (eromano) [inline lib/content-services/index.ts]: NodeDownloadDirective is not needed

File: lib/content-services/permission-manager/components/permission-display/permission-display.component.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    .adf{
+
+        &-display-permission-container {
+            display: flex;
+            justify-content: space-around;
+        }
+
+        &-datatable-permission {
+            display: flex;
+            flex-basis: 38%;
```
Comment (eromano) [inline lib/content-services/permission-manager/components/permission-display/permission-display.component.scss]: not sure it is right flex-basis, how this thing behave in a small screen?

File: lib/content-services/permission-manager/components/permission-display/permission-display.component.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+ * limitations under the License.
+ */
+
+import { Component, ViewEncapsulation, Input, OnInit } from '@angular/core';
+import { NodesApiService } from '@alfresco/adf-core';
+import { MinimalNodeEntryEntity } from 'alfresco-js-api';
+import { PermissionDisplayModel } from '../../models/permission.model';
+
+@Component({
+    selector: 'adf-permission-display',
```
Comment (eromano) [inline lib/content-services/permission-manager/components/permission-display/permission-display.component.ts]: not sure the component name is right, why specify display?

File: lib/content-services/permission-manager/components/permission-display/permission-display.component.scss (hunk 3 above)
Comment (VitoAlbano) [reply lib/content-services/permission-manager/components/permission-display/permission-display.component.scss]: this is just to say to flex : this is the base, we could improve adding the proper flex attribute and chose a behaviour for streched and not but keep in mind that we are basing this on datatable that is not working properly with small screens

File: lib/content-services/permission-manager/components/permission-display/permission-display.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { PermissionDisplayComponent } from './permission-display.component';
+import { NodesApiService } from '@alfresco/adf-core';
+import { Observable } from 'rxjs/Observable';
+
+const fakeNodeWithPermissions: any = {
```
[the remark this answers is not in the document]
Comment (VitoAlbano) [reply lib/content-services/permission-manager/components/permission-display/permission-display.component.spec.ts]: Will add a mock file

**Pull request merged**